reactive/kubernetes_master.py

```diff
@@ -1801,6 +1844,10 @@ def configure_apiserver():
         # No point in trying to start master services and fail. Just return.
         return
 
+    if not is_service_cidr_expansion():
+        hookenv.log("WARN: service cidr changed but not an expansion")
+        return
```

Comment:
This will block all subsequent changes to the apiserver config, rather than simply ignoring the non-expansion CIDR change.